### PR TITLE
feat: add blog post for new v1.0 messaging release

### PR DESCRIPTION
**Pull request by stijnmoreels**
Adds a new quick overview blog post of the recent v1.0 Arcus Messaging release.

Relates to #136

### CODE REVIEW

File: community/blog/2021 Quick overview at the new major Arcus Messaging v1.0 release/README.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    {
+        _logger.LogInformation("Start processing Azure Service Bus message {MessageId}...", message.MessageId);
+    }
+}
+```
+
+Microsoft has also provided a [dedicated migration guide](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/servicebus/Azure.Messaging.ServiceBus/MigrationGuide.md) for this new Azure SDK if you're curious in any new funtionality or changes.
+
+## Extracted message routing for multi-purpose message handling
+Versions before the v1.0 release all used the message pump to both receive and route the messages to the correct registered message handler. In this release, we have made sure that these two resposibilities are split between two services. The message pump still receives the messages, but now a dedicated router will determine and delegate the message to the message handler that can handle the message.
```
Comment (tomkerkhove) [inline community/blog/2021 Quick overview at the new major Arcus Messaging v1.0 release/README.md:42]: What do you think of this? It was a bit much/not fully clear imo
```suggestion
Before v1.0, our message pump was in charge of both receiving and routing messages to the required message handler. 

With this release, we are separating these two responsibilities into dedicated components and laying the groundwork for more powerful message processing.

The message pump is still in charge of receiving messages, but relies on our message router to determine what message handler will process the message.
```

File: community/blog/2021 Quick overview at the new major Arcus Messaging v1.0 release/README.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+We made this change because it allows us to re-use this routing functionality for other purposes. Our ultimate goal is to use this router in Azure Functions where the function trigger takes on the role of the message pump and our message router can be called directly from the function trigger implementation. This would become highly reusable as the exact same message handlers could be used for both approaches: via message pump and via Azure Functions. Currently, the message router cannot be used in Azure Functions (at the time of writing) as we are depending on a [preview package](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.ServiceBus/5.0.0-beta.5).
```
Comment (tomkerkhove) [inline community/blog/2021 Quick overview at the new major Arcus Messaging v1.0 release/README.md:44]:
```suggestion
By doing this, it allows us to re-use our routing functionality and bring it to other message pumps such as Azure Functions.

Your Azure Function will replace the Arcus message pump and route messages to your message handlers accordingly.

This allows you to fully decouple the message pump from your business logic and build re-usable components without a lock-in.

Are you excited about this? So are we! But we need to wait for the official release of [`Microsoft.Azure.WebJobs.Extensions.ServiceBus`](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.ServiceBus/5.0.0-beta.5) first.
```

**Review by tomkerkhove — CHANGES_REQUESTED** [review]
Added a bunch of remarks to improve readability but great post!

File: community/blog/2021 Quick overview at the new major Arcus Messaging v1.0 release/README.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
+# Quick overview at the new major Arcus Messaging v1.0 release
+The new major v1.0 Arcus Messaging release is the biggest one yet: internal improvments, authentication updates, new packages, and many more.
+
+Because the new version contains a lot of breaking changes, we've created a [dedicated migration guide](https://messaging.arcus-azure.net/guides/migration-guide-v1.0) to help consumers in updating towards this new major version.
+
+Come and take a quick look what the release has to offer in this overview blog post.
+
+## New Azure SDK
+Microsoft released a [new client library SDK](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/servicebus/Azure.Messaging.ServiceBus/README.md) for interacting with Azure Service Bus resources. This new library is part of the overall Azure SDK updates. Our [Arcus Security](https://github.com/arcus-azure/arcus.security) library [is already updated](https://github.com/arcus-azure/arcus.security/issues/191) to use this new Azure SDK, so now it was time to update Arcus Messaging.
```
[the remark this answers is not in the document]
Comment (stijnmoreels) [reply community/blog/2021 Quick overview at the new major Arcus Messaging v1.0 release/README.md:9]: Ah, yes, that's better! Thanks!

File: community/blog/2021 Quick overview at the new major Arcus Messaging v1.0 release/README.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+The most notable change here for consumers is the move from the Azure Service Bus `Message` model towards the `ServiceBusReceivedMessage`. This breaking change has the most impact on our fallback message handlers, which receive the full Azure Service Bus message in their implementation.
```
[the remark this answers is not in the document]
Comment (stijnmoreels) [reply community/blog/2021 Quick overview at the new major Arcus Messaging v1.0 release/README.md:13]: Aha, yes!

File: community/blog/2021 Quick overview at the new major Arcus Messaging v1.0 release/README.md (hunk 1 above)
Comment (stijnmoreels) [reply community/blog/2021 Quick overview at the new major Arcus Messaging v1.0 release/README.md:46]: Is great! Bit less technical and more functional which makes this more readable, indeed. Thanks!

File: community/blog/2021 Quick overview at the new major Arcus Messaging v1.0 release/README.md (hunk 2 above)
Comment (stijnmoreels) [reply community/blog/2021 Quick overview at the new major Arcus Messaging v1.0 release/README.md:52]: Is better, for sure.

**Pull request merged**